### PR TITLE
Dynamic meetup pages

### PR DESCRIPTION
**Pull request by alexlbr**
This PR reads the meetups from UM API and creates the meetup pages dynamically on the browser. Ideally, in Gatsbyjs you'd want to build the pages at build time for SEO and meta purposes. Unfortunately, we don't have a way to trigger a build from the backoffice at this point so we'll keep that as a next step.

### CODE REVIEW

File: src/components/buttons/LinkButton.js [hunk 1]
```diff
@@ -9,16 +9,14 @@ import {
...
-// import trackUserBehaviour, { CLICK_ON_CTA } from '../utils/trackUserBehaviour'
 import { ExternalLinkIcon, PdfDownload } from '../../components/icons'
 
 const fontColor = css`
   ${props => {
     const color = props.color
 
     return `
-    text-shadow: 0px 0px 1px ${color} !important;
+    color: ${color};
```
Comment (ric9176) [inline src/components/buttons/LinkButton.js]: Is there no more text shadow on link button? Or it’s coming from styles system now?

**Comment by ric9176** [comment]
Ternary rather than nested if else?

File: src/components/payment/PaymentSection.js [hunk 2]
```diff
@@ -134,10 +136,15 @@ class PaymentSection extends React.Component {
...
     } else {
       title = 'Standard priced ticket'
-      trainingInstanceId = training.id
+      trainingType = training.type
+
+      if (trainingType === MEETUP) {
+        eventId = training.id
+      } else {
+        trainingInstanceId = training.id
+      }
```
Comment (ric9176) [inline src/components/payment/PaymentSection.js]: Ternary rather than nested of else?

Comment (alexlbr) [reply src/components/payment/PaymentSection.js]: @ric9176 we are assigning two different variables eventId and trainingInstanceId, so I think we can't use the ternary

File: src/components/buttons/LinkButton.js (hunk 1 above)
Comment (alexlbr) [reply src/components/buttons/LinkButton.js]: @ric9176 just to double check it refers to text-shadow, not to box-shadow, so buttons have still shadow. This one made the bold text inside the buttons harder to read. I think this is something Horacio had pointed out a few weeks ago about some text being blurred. I've also noticed it today so I fixed it

**Pull request merged**